src/Lecture2.hs

```diff
@@ -181,7 +262,10 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing (_:[]) = True
+isIncreasing (a:b:list) = a <= b && isIncreasing list
+                       -- a <  b passed test cases, but shouldn't've I think
```

Comment:
Wel, the intended behaviour is to use `a < b`. The function is called `isIncreasing` and not `isSorted` or `isNonDecreasing` 😉